static/front/peek.js

```diff
@@ -528,6 +569,13 @@ $(function() {
 
     $(window).on("hashchange", function() {
       $(document.body).toggleClass("peek", window.location.hash == "#p");
```

Comment:
Looks good overall, just the one small fix. There are also some merge conflicts that will need to be worked out before it can be merged.